render.go

```diff
@@ -185,8 +184,8 @@ func (*RenderSystem) Type() string {
 	return "RenderSystem"
 }
 
-func (rs *RenderSystem) Priority() int {
-	return 1
+func (*RenderSystem) Priority() int {
+	return -10000 // some low value
```

Comment:
perhaps it is better to use some constants, like Integer.MIN_VAL, does something like this exists in golang?